ci-operator/step-registry/ipi/conf/aws/min-perms-user/ipi-conf-aws-min-perms-user-ref.yaml

```diff
@@ -0,0 +1,21 @@
+ref:
+  as: ipi-conf-aws-min-perms-user
+  from_image:
+    namespace: ocp
+    name: "4.18"
```

Comment:
Because the step has dependency of installer, sounds like we can not use specific version tag of upi-installer image, but use `from: upi-installer`, so that the step can used is different versions of OCP ci jobs.

Reply:
The QE step uses `name: "4.12"`. So you think I can remove the `name: "4.18"` part?

Reply:
I meant for this step, we should use:
```
from: upi-installer
```
instead of:
```
from_image:
    namespace: ocp
    name: "4.18"
    tag: upi-installer
```
So that the step would rely on upi-installer base image setting in ci yaml file.

E.g:  upi-installer base image is supposed to be 4.19 in 4.19 ci yaml file, , then for this step, it would use 4.19 installer to run `create permissions-policy`, but not use a fixed version of upi-installer to run this step.

Reply:
Fair enough. I'll change it.